internal/config/config.go

```diff
@@ -132,6 +154,9 @@ func (c *Config) checkNode() error {
 				return fmt.Errorf("unknown error checking if node socket path exists: %s", err)
 			}
 		}
```

Comment:
We can probably get rid of this check since we're actually connecting to the socket now

Reply:
Probably. I'll get rid of it next iteration.